tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -0,0 +1,462 @@
+          alternates?: Array<string>;
+        }
+      | {
+          obscured: ObscuringLevel;
+        }
+    ) &
+      Partial<Interactable>);
+
+type StripMenuData = {
+  items: Record<keyof typeof SLOTS, StripMenuItem>;
+  name: string;
+};
+
+export const StripMenu = (props, context) => {
+  const { act, data } = useBackend<StripMenuData>(context);
+
+  const gridSpots = new Map<GridSpotKey, string>();
+  for (const key of Object.keys(data.items)) {
+    gridSpots.set(SLOTS[key].gridSpot, key);
+  }
+
+  const get_button_color = (item) => {
+    if (!item) {
+      return 'translucent';
+    }
+    if (item.cantstrip) {
+      return 'transparent';
+    }
+    if (item.interacting) {
+      return 'average';
+    }
+    return 'translucent';
+  };
+
+  return (
+    <Window
+      title={`Stripping ${data.name}`}
+      width={360}
+      height={390}
+      theme="nologo"
+    >
+      <Window.Content style={{ 'background-color': 'rgba(0, 0, 0, 0.5)' }}>
+        <Stack fill vertical>
+          {range(0, ROWS).map((row) => (
+            <Stack.Item key={row}>
+              <Stack fill>
+                {range(0, COLUMNS).map((column) => {
+                  const key = getGridSpotKey([row, column]);
+                  const keyAtSpot = gridSpots.get(key);
+
+                  if (!keyAtSpot) {
+                    return (
+                      <Stack.Item
+                        key={key}
+                        style={{
+                          width: BUTTON_DIMENSIONS,
+                          height: BUTTON_DIMENSIONS,
+                        }}
+                      />
+                    );
+                  }
+
+                  const item = data.items[keyAtSpot];
+                  const slot = SLOTS[keyAtSpot];
+
+                  let alternateActions: Array<string> | undefined;
+
+                  let content;
+                  let tooltip;
+
+                  if (item === null) {
+                    tooltip = slot.displayName;
+                  } else if ('name' in item) {
+                    content = (
+                      <Box
+                        as="img"
+                        src={`data:image/jpeg;base64,${item.icon}`}
+                        height="100%"
+                        width="100%"
+                        style={{
+                          '-ms-interpolation-mode': 'nearest-neighbor',
+                          'vertical-align': 'middle',
+                        }}
+                      />
+                    );
+
+                    tooltip = item.name;
+                  } else if ('obscured' in item) {
+                    content = (
+                      <Icon
+                        name={
+                          item.obscured === ObscuringLevel.Completely
+                            ? 'ban'
+                            : 'eye-slash'
+                        }
+                        size={3}
+                        ml={0}
+                        mt={2.75}
```

Comment:
```suggestion
                        mt={2.5}
```
Icon is slightly off-centre